src/backends/ncnn/ncnnlib.cc

```diff
@@ -85,7 +85,10 @@ namespace dd
   NCNNLib<TInputConnectorStrategy, TOutputConnectorStrategy,
           TMLModel>::~NCNNLib()
   {
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdelete-non-virtual-dtor"
```

Comment:
Do you know where this warning comes from?

Reply:
somthing like that :```